### ISSUE TITLE
train/test split

### ISSUE THREAD

**Issue by xwhjy** [RoySheffer/im2wav#5]
Hi,

I was wondering in your experiments did you use the exact same train/test split as specified in VGGSound's csv file or did you simply just use a train_test_split ratio to divide the whole dataset. Because when I checked the code, I saw you did the split [here](https://github.com/RoySheffer/im2wav/blob/aff5a8022a01760052f633ec525e36d1cdf70403/models/data/data_processor.py#L40) using the ratio. But in the paper, you mentioned that "We follow the original VGGSound train/test splits." Please correct me if I was wrong. Thank you!

**Comment by RoySheffer** [comment]
We used the same train/test split as specified in VGGSound.
The code you mentioned is used for the "during-training testing" AKA validation set taken from the VGGSound training set. The VGGSound Test set is not present in any way during training.
